include/z64scene.h

```diff
@@ -534,119 +534,119 @@ typedef union {
-    /* 0x44 */ SCENE_HAKUGIN_BS,
-    /* 0x45 */ SCENE_20SICHITAI,
-    /* 0x46 */ SCENE_21MITURINMAE,
-    /* 0x47 */ SCENE_LAST_ZORA,
-    /* 0x48 */ SCENE_11GORONNOSATO2,
-    /* 0x49 */ SCENE_SEA,
-    /* 0x4A */ SCENE_35TAKI,
-    /* 0x4B */ SCENE_REDEAD,
-    /* 0x4C */ SCENE_BANDROOM,
-    /* 0x4D */ SCENE_11GORONNOSATO,
-    /* 0x4E */ SCENE_GORON_HAKA,
-    /* 0x4F */ SCENE_SECOM,
-    /* 0x50 */ SCENE_10YUKIYAMANOMURA,
-    /* 0x51 */ SCENE_TOUGITES,
-    /* 0x52 */ SCENE_DANPEI,
-    /* 0x53 */ SCENE_IKANAMAE,
-    /* 0x54 */ SCENE_DOUJOU,
-    /* 0x55 */ SCENE_MUSICHOUSE,
-    /* 0x56 */ SCENE_IKNINSIDE,
-    /* 0x57 */ SCENE_MAP_SHOP,
-    /* 0x58 */ SCENE_F40,
-    /* 0x59 */ SCENE_F41,
-    /* 0x5A */ SCENE_10YUKIYAMANOMURA2,
-    /* 0x5B */ SCENE_14YUKIDAMANOMITI,
-    /* 0x5C */ SCENE_12HAKUGINMAE,
-    /* 0x5D */ SCENE_17SETUGEN,
-    /* 0x5E */ SCENE_17SETUGEN2,
-    /* 0x5F */ SCENE_SEA_BS,
-    /* 0x60 */ SCENE_RANDOM,
-    /* 0x61 */ SCENE_YADOYA,
-    /* 0x62 */ SCENE_KONPEKI_ENT,
-    /* 0x63 */ SCENE_INSIDETOWER,
-    /* 0x64 */ SCENE_26SARUNOMORI,
-    /* 0x65 */ SCENE_LOST_WOODS,
-    /* 0x66 */ SCENE_LAST_LINK,
-    /* 0x67 */ SCENE_SOUGEN,
-    /* 0x68 */ SCENE_BOMYA,
-    /* 0x69 */ SCENE_KYOJINNOMA,
-    /* 0x6A */ SCENE_KOEPONARACE,
-    /* 0x6B */ SCENE_GORONRACE,
-    /* 0x6C */ SCENE_TOWN,
-    /* 0x6D */ SCENE_ICHIBA,
-    /* 0x6E */ SCENE_BACKTOWN,
-    /* 0x6F */ SCENE_CLOCKTOWER,
-    /* 0x70 */ SCENE_ALLEY,
+    /* 0x3B */ SCENE_TORIDE, // Pirates' Fortress Moat
+    /* 0x3C */ SCENE_FISHERMAN, // Fisherman's Hut
+    /* 0x3D */ SCENE_GORONSHOP, // Goron Shop
+    /* 0x3E */ SCENE_DEKU_KING, // Deku King's Chamber
+    /* 0x3F */ SCENE_LAST_GORON, // Goron Trial
+    /* 0x40 */ SCENE_24KEMONOMITI, // Road to Southern Swamp
+    /* 0x41 */ SCENE_F01_B, // Doggy Racetrack
+    /* 0x42 */ SCENE_F01C, // Cucco Shack
+    /* 0x43 */ SCENE_BOTI, // Ikana Graveyard
+    /* 0x44 */ SCENE_HAKUGIN_BS, // Goht's Lair
+    /* 0x45 */ SCENE_20SICHITAI, // Southern Swamp (poison)
+    /* 0x46 */ SCENE_21MITURINMAE, // Woodfall
+    /* 0x47 */ SCENE_LAST_ZORA, // Zora Trial
+    /* 0x48 */ SCENE_11GORONNOSATO2, // Goron Village (spring)
+    /* 0x49 */ SCENE_SEA, // Great Bay Temple
+    /* 0x4A */ SCENE_35TAKI, // Waterfall Rapids
+    /* 0x4B */ SCENE_REDEAD, // Beneath the Well
+    /* 0x4C */ SCENE_BANDROOM, // Zora Hall Rooms
+    /* 0x4D */ SCENE_11GORONNOSATO, // Goron Village (winter)
+    /* 0x4E */ SCENE_GORON_HAKA, // Goron Graveyard
+    /* 0x4F */ SCENE_SECOM, // Sakon's Hideout
+    /* 0x50 */ SCENE_10YUKIYAMANOMURA, // Mountain Village (winter)
+    /* 0x51 */ SCENE_TOUGITES, // Ghost Hut
+    /* 0x52 */ SCENE_DANPEI, // Deku Shrine
+    /* 0x53 */ SCENE_IKANAMAE, // Road to Ikana
+    /* 0x54 */ SCENE_DOUJOU, // Swordsman's School
+    /* 0x55 */ SCENE_MUSICHOUSE, // Music Box House
+    /* 0x56 */ SCENE_IKNINSIDE, // Igos du Ikana's Lair
+    /* 0x57 */ SCENE_MAP_SHOP, // Tourist Information
```

Comment:
```suggestion
    /* 0x57 */ SCENE_MAP_SHOP, // Southern Swamp Tourist Information
```
Nowhere else in the game would probably fit the old description, but to help narrow on first read